### DISCUSSION TITLE
How can i record a voicechat?

### DISCUSSION THREAD

**Discussion by LordFortyTwo** [Rapptz/discord.py#6922] [Q&A]
Hi, i want to record a voicechat and found *vc.start_recording* but i get following error: *AttributeError: 'VoiceClient' object has no attribute 'start_recording'*
Any idea what i can do?

**Comment by tk421bsod** [comment]
This isn't implemented in this library yet, but [a fork](https://github.com/Sheepposu/discord.py) implements it. See #6507 for more information.
